internal/pkg/controller/controller_test.go

```diff
@@ -336,7 +336,7 @@ func TestControllerForUpdatingConfigmapShouldUpdateDeployment(t *testing.T) {
 }
 
 // Do not Perform rolling upgrade on deployment and create env var upon updating the labels configmap
-func TestControllerUpdatingConfigmapLabelsShouldNotCreateorUpdateEnvInDeployment(t *testing.T) {
+func TestControllerUpdatingConfigmapLabelsShouldNotCreateOrUpdateEnvInDeployment(t *testing.T) {
```

Comment:
Almost every word in this very long function name is capitalized. *except* for `or`. (`Configmap` is a a word in this repo.) `Createor` looks too close to a typo for `Creator` and there's really no harm in capitalizing the letter `o`.